com.github.zadam.trilium.yml

```
@@ -0,0 +1,96 @@
+app-id: com.github.zadam.trilium
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: trilium
+
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

Not a fallback anymore without Wayland socket.